app/views/home/how_government_works.html.erb

```diff
@@ -3,18 +3,24 @@
 <% meta_description "In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out who runs government and how government is run, as well as learning about the history of government." %>
 
 <div id="how-government-works">
-  <header class="block headings-block">
-    <div class="inner-block floated-children">
-      <%= render partial: 'shared/heading',
-                locals: { big: true,
-                          heading: "How government works" } %>
-      <p class="intro-paragraph">In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out <a href="#who-runs-government">who runs government</a> and <a href="#how-government-is-run">how government is run</a>, as well as learning about the <a href="#history-uk-government">history of government</a>.</p>
+  <header class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-three-quarters">
+        <%= render "govuk_publishing_components/components/title", {
+          title: "How government works",
+        } %>
+        <p class="govuk-body-l">In the UK, the Prime Minister leads the government with the support of the Cabinet and ministers. You can find out <a class="govuk-link" href="#who-runs-government">who runs government</a> and <a class="govuk-link" href="#how-government-is-run">how government is run</a>, as well as learning about the <a class="govuk-link" href="#history-uk-government">history of government</a>.</p>
+    </div>
     </div>
   </header>
 
   <div class="block">
     <div class="inner-block">
-      <h2 class="coalition" id="who-runs-government">Who runs government</h2>
+      <%= render "govuk_publishing_components/components/heading", {
```

Comment:
This was rendered with a bottom border. We should add an `<hr>` in to keep the design of the template consistent.

Also the styling of this h2 is exactly the same as the h3 below for "The Prime Minister"

Reply:
The styling is a limitation of the component. I'll undo all the components and revert most of the changes here.